### PR TITLE
fix: connectors worker logs on DD

### PR DESCRIPTION
**Pull request by fontanierh**

### CODE REVIEW

File: k8s/configmaps/connectors-worker-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (fontanierh) [inline k8s/configmaps/connectors-worker-configmap.yaml:1]: most likely useless

File: k8s/deployments/connectors-worker-deployment.yaml [hunk 2]
```diff
@@ -14,16 +14,16 @@ spec:
         name: connectors-worker-pod
         admission.datadoghq.com/enabled: "true"
       annotations:
-        ad.datadoghq.com/web.logs: '[{"source": "connectors-worker","service": "connectors","tags": ["env:prod"]}]'
+        ad.datadoghq.com/web.logs: '[{"source": "connectors-worker","service": "connectors-worker","tags": ["env:prod"]}]'
     spec:
       containers:
-        - name: worker
+        - name: web
```
Comment (fontanierh) [inline k8s/deployments/connectors-worker-deployment.yaml:20]: this is the actual fix afaict

File: k8s/services/connectors-worker-service.yaml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
```
Comment (fontanierh) [inline k8s/services/connectors-worker-service.yaml:1]: probably useless

**Review by spolu — APPROVED** [review]
LGTM

**Pull request merged**